### PR TITLE
Fix/typos

### PR DESCRIPTION
**Pull request by alliefleming**
Just general content changes and typo fixes. Looks like we have a page for /interns and /internships – is it necessary to have both?

Also curious why we can't automatically merge and how to resolve that.

### CODE REVIEW

File: locations/fullerton/space-features.html [hunk 1]
```diff
@@ -69,7 +69,7 @@ <h3 class="card-title">Drip coffee, cold brew &amp; kombucha on tap</h3>
     </div>
     <div class="card-block">
       <h3 class="card-title">Community Events</h3>
-      <p class="card-text">As a member, your perks extend beyond the work place and into the Fullerton community. You will have free access to member-exclusive events like Forum where you will learn from social good pioneers right here at our Costa Mesa location. Members also enjoy discounts on select, ticketed events and workshops.</p>
+      <p class="card-text">As a member, your perks extend beyond the work place and into the Fullerton Community. You will have free access to member-exclusive events like Forum, where you will learn from social good pioneers right here at our Fullerton location. Members also enjoy discounts on select, ticketed Wayfare events and workshops.</p>
```
Comment (brandonarbini) [inline locations/fullerton/space-features.html]: I don't think `community` should be capitalized.

File: internships.markdown [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-title: Interships
+title: Internships
 position: 99
 redirect_from: "/interns/index.html"
 ---
 
 <img src="/assets/images/wayfare-interns.jpg" class="img-fluid">
 
-## Official Program Details + Dates
+## Official program details & dates
```
Comment (brandonarbini) [inline internships.markdown:9]: I don't understand the various decapitalizations (not a word) here.

**Review by brandonarbini — CHANGES_REQUESTED** [review]
Feel free to deploy this an we can tweak things further later.

File: locations/fullerton/space-features.html (hunk 1 above)
Comment (alliefleming) [reply locations/fullerton/space-features.html]: will un-capitalize that. good find.

**Pull request merged**